xml/security_vpnserver.xml

```diff
@@ -621,9 +621,12 @@ PRE_DOWN_SCRIPT='systemd:openvpn@client'</screen>
     Before a VPN connection can be established, the client must authenticate
     the server certificate. Conversely, the server must also authenticate
     the client certificate. This is called <emphasis>mutual
-    authentication</emphasis>. To create such certificates, use the
+    authentication</emphasis>.
+    <!-- toms 2019-09-06: Disabled as there is no YaST CA module.
+    To create such certificates, use the
     &yast; CA module. See <xref linkend="cha-security-yast-ca"/> for more
     details.
+    -->
    </para>
    <para>
     To create a VPN root, server, and client CA, proceed as follows:
```

Comment:
The procedure that follows needs to be reworked, too. Just assume the certificate was created elsewhere.

Reply:
Thanks for the feedback! :+1: Hmn, I think, this is not that easy.

It seems, "rework" means basically "remove", because the steps in procedure ID=pro-security-vpn-serverca are completely related to the YaST CA module. Same with the next procedure (starting with step "Choose the Certificate tab"); it exports the VPN server and client certificates into PEM format.

I see currently no way how to replace it with something else. But maybe I'm too tired for today, so we can discuss this next week. :wink:

Reply:
Submitted resolution for this with b72b023